src/Microsoft.Azure.WebJobs.Host/Utility.cs

```diff
@@ -35,6 +36,13 @@ internal static string GetFunctionName(MethodInfo methodInfo)
             return (functionNameAttribute != null) ? functionNameAttribute.Name : methodInfo.Name;
         }
 
+        internal static string GetFunctionShortName(MethodInfo methodInfo)
+        {
+            FunctionNameAttribute functionNameAttribute = TypeUtility.GetHierarchicalAttributeOrNull<FunctionNameAttribute>(methodInfo);
+            var methodName = String.Format(CultureInfo.InvariantCulture, "{0}.{1}", methodInfo.DeclaringType.Name, methodInfo.Name);
+            return (functionNameAttribute != null) ? functionNameAttribute.Name : methodName;
```

Comment:
Can you call `methodInfo.GetShortName()` here so we're guaranteed to be consistent? It'll prevent building that string unless it's required, too. Like:
```
return (functionNameAttribute != null) ? functionNameAttribute.Name : methodInfo.GetShortName();
```